### ISSUE TITLE
Usernames are broken

### ISSUE THREAD

**Issue by tox-user** [qTox/qTox#5013]
Aliases in qTox have been broken for a long time (maybe forever). They would always be set with every name change of a friend and they were set to their new username. The result was that in addition to updating friend's name, we would also set their alias to their new name. Such behavior was impossible to notice, because of how qTox displays aliases - they replace a friend's name. There was also a second issue. When a friend changed their name, we would only see the changed name on friends list, but not in group chat.

PR https://github.com/qTox/qTox/pull/4970 was supposed to fix both of those problems and it almost did. It couldn't, because the previous broken behavior of aliases caused everyone's friends to have an alias set and the PR made aliases not reset with name change anymore. That means everyone running qTox 1.14.0 can't see name changes of their friends. They are blocked from seeing them, because of aliases that were automatically set for everyone. Most people do not realize they were using aliases without knowing. Usernames are simply broken now.

This problem is not easy to fix, but I propose we bring back the previous behavior of aliases for 1.14.1 release to make names usable for now. Only this time we should make aliases just reset on name change and not change them to a new name of a user. We should also add a new variable "wasAliasSet" to each friend, which would let us determine if an alias was automatic (set by previous broken behavior) or set by the user. In result we could reset all old, automatically set aliases in a future release with a little less damage to users.

So the plan would be to:
1. Make aliases reset (set them to an empty string) on name change in 1.14.1
2. Reset all automatic aliases in 1.15 or later and make user aliases stick forever - like they do now in 1.14

**Comment by anthonybilinski** [comment]
I agree the behaviour of users not seeing name changes in v1.14.0 is broken. The complete solution would probably be to add some config file bool of "was this alias manually set by user" for new versions, if that's not set to true reset their alias and show the name, when the user sets a manual alias then set it to true and save the alias separate from the friend name.

I'm ok with "Make aliases reset (set them to an empty string) on name change in 1.14.1" since that should just be a 1-ish line change and will keep behaviour consistent with previous releases, where a friend changing their name resets your "alias". I don't want to include more than that into 1.14.1 since I want to get it out ASAP due to multiple major crash fixes after 1.14.0, so want to keep testing/complexity of change minimal.
